src/WebJobs.Script/Description/FunctionDescriptorProvider.cs

```diff
@@ -104,6 +104,9 @@ public virtual bool TryCreate(FunctionMetadata functionMetadata, out FunctionDes
             // Add ExecutionContext to provide access to InvocationId, etc.
             parameters.Add(new ParameterDescriptor(ScriptConstants.SystemExecutionContextParameterName, typeof(ExecutionContext)));
 
+            // Add the CancellationToken. We need this to ensure we can cancel even if the Function hasn't requested it.
+            parameters.Add(new ParameterDescriptor(ScriptConstants.SystemCancellationTokenParameterName, typeof(CancellationToken)));
```

Comment:
This injection of a cancellation token isn't needed anymore, right? as per our discussions